actions/infer_bundle_identifier.rb

```diff
@@ -17,20 +17,12 @@ class InferBundleIdentifierAction < Action
         production: "com.%s.%s"
       }.freeze
 
-      # If the application is deployed in Wolox account choose Wolox
-      # as owner, otherwise choose project name.
-      DEPLOYED_IN_WOLOX_ACCOUNT = {
-        test: true,
-        qa: true,
-        appstore: true,
-        production: false
-      }.freeze
-
       def self.run(params)
         # For legacy projects, just override the return value
         # with the desired bundle identifier.
         bundle_identifier_format = BUNDLE_IDENTIFIERS_FORMAT[params[:build_configuration]]
-        team_name = DEPLOYED_IN_WOLOX_ACCOUNT[params[:build_configuration]] ? "Wolox" : ProjectNameAction.default_project_name
+        uses_internal_account = Actions::UsesInternalAccountAction.run(params[:build_configuration])
+        team_name = uses_internal_account ? "Wolox" : ProjectNameAction.default_project_name
```

Comment:
I know the name will always be `Wolox` but I don't like this hardcoding. Shouldn't we take it from the project or something?

Reply:
I'll move it to a constant at least.